ratpack/src/main/java/brave/example/Frontend.java

```diff
@@ -0,0 +1,53 @@
+package brave.example;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.inject.name.Names;
+import java.net.URI;
+import javax.inject.Inject;
+import javax.inject.Named;
+import ratpack.exec.Promise;
+import ratpack.guice.Guice;
+import ratpack.http.client.HttpClient;
+import ratpack.server.RatpackServer;
+import ratpack.server.ServerConfig;
+import ratpack.zipkin.ServerTracingModule;
+import ratpack.zipkin.Zipkin;
+
+public final class Frontend {
+  final HttpClient client;
+  final URI backendEndpoint;
+
+  @Inject Frontend(@Zipkin HttpClient client, @Named("backendEndpoint") String backendEndpoint) {
```

Comment:
I don't know if there's a more ratpack idiomatic way to get a single string value from config..